gap/examples.gi

```diff
@@ -714,3 +714,41 @@ function(filt, m, n)
   SetIsSymmetricDigraph(D, true);
   return D;
 end);
+
+InstallMethod(BookDigraphCons,
+  "for IsMutableDigraph and one positive integer",
+  [IsMutableDigraph, IsPosInt],
+  function(filt, m)
+    local pages, book, graph;
+    # special cases
+    if m = 1 then
+      # "the" square graph
+      return CompleteBipartiteDigraph(filt, 2, 2);
+    elif m = 2 then
+      # domino graph
+      return SquareGridGraph(filt, 2, 3);
+    fi;
+    pages := StarDigraph(filt, m);
```

Comment:
Also, as noted above, this does not match the definition on Wolfram MathWorld. It's the star digraph on m+1 vertices, not m.